plugin/forward/health.go

```diff
@@ -40,7 +45,14 @@ func (h *dnsHc) SetTLSConfig(cfg *tls.Config) {
 	h.c.TLSConfig = cfg
 }
 
-// For HC we send to . IN NS +norec message to the upstream. Dial timeouts and empty
+func (h *dnsHc) SetRecursionDesired(recursionDesired bool) {
+	h.recursionDesired = recursionDesired
+}
+func (h *dnsHc) GetRecursionDesired() bool {
```

Comment:
Drop Get from this - do we really need it btw?

Reply:
It does feel a bity java'y. But i don't know how else to get the property out of the dnsHc-instance. Any tips?